en/reports/vivliostyle-report-2020/vf2020report.md

```diff
@@ -0,0 +1,261 @@
+---
+title: "Vivliostyle Foundation FY2020 Activity Report"
+lang: en
+---
+
+## **Chapter 1: Activity Report for FY2020**
+
+(The 3rd Fiscal Year: from April 1, 2020 to March 31, 2021)
+
+### 1. Organize developer/user events
+
+We have decided to hold developer/user events twice a year as a place to widely let users and developers know about Vivliostyle. This year, we were able to hold the following events.
+
+- [April 4, 2020: Vivliostyle User/Dev Meetup 2020 Spring (Held online)](https://vivliostyle.connpass.com/event/170939/)
+- [October 24, 2020: Vivliostyle User/Dev Meetup 2020 Autumn (Held online)](https://vivliostyle.connpass.com/event/189940/)
+
+The spring event coincided with the COVID 19 calamity. Initially, the event was scheduled to be held at the Japan Association of Graphic Arts Technology (JAGAT), but as Avoid the “Three Cs” was becoming a top priority, we were able to get through it by switching to online distribution. While most of the similar events were cancelled, thankfully, 142 people registered for the event. In terms of content, we were able to inform people about the real activities of our corporation, focusing on the report of Vivliostyle Pub, which was in the midst of alpha development at the time.
+
+On the other hand, the fall event was a thoughtful seminar on CSS typesetting. For example, [Simon Worthington](https://www.force11.org/users/simon-worthington)of the Open Science Lab - German National Library of Science and Technology reported on Rapid publishing for public health books against COVID 19. And our contributor [akabeko](https://github.com/akabekobeko) reported on Vivliostyle's competitor [Paged.js](https://www.pagedjs.org/). In this way, we were able to show the appeal of CSS typesetting beyond Vivliostyle.
```

Comment:
What do you mean by "thoughtful"?

Reply:
I see, it is indeed a difficult expression to understand. How about the following?

On the other hand, the autumn event seemed to be a good opportunity to work on CSS typesetting as a seminar on the subject.

Reply:
Sounds good